tests/integration/ha_test.go

```diff
@@ -124,18 +124,21 @@ func TestInitWithMultipleKeepers(t *testing.T) {
 }
 
 func setupServers(t *testing.T, clusterName, dir string, numKeepers, numSentinels uint8, syncRepl bool, usePgrewind bool, primaryKeeper *TestKeeper) (testKeepers, testSentinels, *TestProxy, *TestStore) {
-	var initialClusterSpec *cluster.ClusterSpec
+	var initialClusterSpec = &cluster.ClusterSpec{
```

Comment:
Small refactor here to put the common ClusterSpec up top, then assign the overrides. It might make sense to use something like https://github.com/imdario/mergo in future, so we can merge optional overrides provided in this `setupServers` helper.